lib/scrub/max_ocn.rb

```diff
@@ -45,17 +49,25 @@ def ocn
 
     def write_file
       @log.info "writing new memo file"
-      res = `curl -s "#{OCLC_URL}"`
-      data = JSON.parse(res)
-      f = File.open(MEMO_LOC, "w")
+      data = JSON.parse(make_call)
+      f = File.open(MaxOcn.memo_loc, "w")
       @ocn = data["oclcNumber"].to_i
       f.puts @ocn
       f.close
       @ocn
     end
 
+    def make_call
+      if @mock
+        @log.info "mock call"
+        `cat spec/fixtures/max_oclc_response.json`
+      else
+        `curl -s "#{OCLC_URL}"`
```

Comment:
Simplest possible way of doing this in ruby that I'm aware of is:
```
require 'open-uri'

URI.open(OCLC_URL).read
```

Reply:
yep